app/templates/buyers/award_or_cancel_brief.html

```diff
@@ -7,9 +7,34 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {% with items = breadcrumbs %}
-    {% include "toolkit/breadcrumb.html" %}
-  {% endwith %}
+  {{ govukBreadcrumbs({
+    "items": [
+        {
+          "href": "/",
+          "text": "Digital Marketplace"
+      },
+      {
+          "href": url_for("buyers.buyer_dashboard"),
+          "text": "Your account"
+      },
+      {
+          "href": url_for("buyers.buyer_dos_requirements"),
+          "text": "Your requirements"
+      },
+      {
+        "href": url_for(
+          ".view_brief_overview",
+          framework_slug=brief['frameworkSlug'],
+          lot_slug=brief['lotSlug'],
+          brief_id=brief['id']
+        ),
+        "text": brief['title']
+      },
+      {
+          "text": "Requirements already updated for " + brief.title
```

Comment:
This seems like an error message to me?

Reply:
Done.

Reply:
This could be `Have you awarded a contract?`.

We don't need the requirement name in this final breadcrumb, as it is already in the previous breadcrumb.

![image](https://user-images.githubusercontent.com/503614/72894768-ea222280-3d13-11ea-9657-d09262b7cab1.png)

Reply:
Changed.